persistence/schema/fisherman.go

```diff
@@ -0,0 +1,29 @@
+package schema
```

Comment:
For this file and all the others, I wanted to add a point about migrations & schemas that I feel pretty strongly about before moving forward.

I looked at different [SQL migration tools](https://povilasv.me/go-schema-migration-tools/) in Golang and though I haven't picked a specific one, [golang-migrate/migrate](https://github.com/golang-migrate/migrate) seems to be the most popular one. However, instead of defining our tables in strings withing `.go` file, we should follow the best practices for how to manage migrations (note that creation is a migration in itself).

Source: golang-migrate/migrate
tl;dr
1. We store these in `.sql` files
2. Each change will have a `<date>_<migration>.up.sql` and a `<date>_<migration>.down.sql` file
3. This will enable:
- Cleaner & easier to read code (separation of go and SQL)
- Going back & forth during development
- Easy to track changes / migrations when we actually do them on chain.

Reply:
https://github.com/pokt-network/pocket/issues/77